app/_includes/md/kgo/konnect-entities-prerequisites.md

```diff
@@ -0,0 +1,133 @@
+{% unless include.disable_accordian %}
+<details class="custom" markdown="1">
+<summary>
+<blockquote class="note">
+  <p style="cursor: pointer">Before you create any Konnect entity, make sure you've <u>installed {{site.kgo_product_name}} and created a valid KonnectAPIAuthConfiguration{% if include.with-control-plane %} and KonnectGatewayControlPlane{% endif %}</u> in your cluster.</p>
+</blockquote>
+</summary>
+
+## Prerequisites
+{% endunless %}
+
+{% include md/kgo/prerequisites.md disable_accordian=true version=page.version release=page.release kconf-crds=true %}
+
+### Create an access token in Konnect
+
+You may create either a Personal Access Token (PAT) or a Service Account Token (SAT) in Konnect. Please refer to the
+[Konnect authentication documentation](/konnect/api/#authentication) for more information. You will need this token
+to create a `KonnectAPIAuthConfiguration` object that will be used by the {{site.kgo_product_name}} to authenticate
+with Konnect APIs.
+
+### Create a Konnect API Auth Configuration
+
+Depending on your preferences, you might want to create a `KonnectAPIAuthConfiguration` object with the token specified
+directly in its spec or as a reference to a Kubernetes Secret. The `serverURL` field should be set to the Konnect API
```

Comment:
```suggestion
directly in its spec or as a reference to a Kubernetes secret. The `serverURL` field should be set to the Konnect API
```

Reply:
I think we should stick to upper case as that's a Kubernetes resource name.